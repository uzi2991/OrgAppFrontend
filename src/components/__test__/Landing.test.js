import React from "react";
import renderer from "react-test-renderer";
import { BrowserRouter as Router } from 'react-router-dom';
import "@testing-library/jest-dom";
import Landing from "../../pages/Landing";

test("Landing snapshot", () => {
    const landingSnapshot = renderer.create(
        <Router>
            <Landing />
        </Router>

    ).toJSON();

    expect(landingSnapshot).toMatchSnapshot();
})

