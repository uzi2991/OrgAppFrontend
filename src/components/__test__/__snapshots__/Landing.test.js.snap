// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Landing snapshot 1`] = `
<div
  className="landing-banner"
>
  <img
    className="landing-banner__image"
    src="bg1.jpg"
  />
  <div
    className="landing-banner__content"
  >
    <h1
      className="landing-banner__title"
    >
      Trello lets you work more collaboratively and get more done
    </h1>
    <h4
      className="landing-banner__subtitle"
    >
      Trello's boards, lists, and cards enable you to organize and prioritize your projects in a fun, flexible, and rewarding way.
    </h4>
    <a
      className="btn"
      href="/register"
      onClick={[Function]}
    >
      Sign Up For Free
    </a>
  </div>
</div>
`;
